autogen/logger/sqlite_logger.py

```diff
@@ -0,0 +1,303 @@
+                CREATE TABLE IF NOT EXISTS agents (
+                    id INTEGER PRIMARY KEY,                             -- Key assigned by the database
+                    agent_id INTEGER,                                   -- result of python id(agent)
+                    wrapper_id INTEGER,                                 -- result of python id(agent.client)
+                    session_id TEXT,
+                    name TEXT,                                          -- agent.name
+                    class TEXT,                                         -- type or class name of agent
+                    init_args TEXT,                                     -- JSON serialization of constructor
+                    timestamp DATETIME DEFAULT CURRENT_TIMESTAMP,
+                    UNIQUE(agent_id, session_id))
+            """
+            self.cur.execute(query)
+            self.con.commit()
+
+            query = """
+                CREATE TABLE IF NOT EXISTS oai_wrappers (
+                    id INTEGER PRIMARY KEY,                             -- Key assigned by the database
+                    wrapper_id INTEGER,                                 -- result of python id(wrapper)
+                    session_id TEXT,
+                    init_args TEXT,                                     -- JSON serialization of constructor
+                    timestamp DATETIME DEFAULT CURRENT_TIMESTAMP,
+                    UNIQUE(wrapper_id, session_id))
+            """
+            self.cur.execute(query)
+            self.con.commit()
+
+            query = """
+                CREATE TABLE IF NOT EXISTS oai_clients (
+                    id INTEGER PRIMARY KEY,                             -- Key assigned by the database
+                    client_id INTEGER,                                  -- result of python id(client)
+                    wrapper_id INTEGER,                                 -- result of python id(wrapper)
+                    session_id TEXT,
+                    class TEXT,                                         -- type or class name of client
+                    init_args TEXT,                                     -- JSON serialization of constructor
+                    timestamp DATETIME DEFAULT CURRENT_TIMESTAMP,
+                    UNIQUE(client_id, session_id))
+            """
+            self.cur.execute(query)
+            self.con.commit()
+
+            query = """
+            CREATE TABLE IF NOT EXISTS version (
+                id INTEGER PRIMARY KEY CHECK (id = 1),                  -- id of the telemetry database
+                version_number INTEGER NOT NULL                         -- version of the telemetry database
+            );
+            """
+            self.cur.execute(query)
+            self.con.commit()
+
+            current_verion = self._get_current_db_version()
+            if current_verion is None:
+                query = "INSERT INTO version (id, version_number) VALUES (1, 1);"
```

Comment:
We can make a static variable in this module `schema_version = 1` and use that in this `INSERT`.